### PR TITLE
SAK-43024 Forums / Automatically create multiple topics for group issue

### PR DESCRIPTION
**Pull request by mpellicer**
https://jira.sakaiproject.org/browse/SAK-43024

### CODE REVIEW

File: msgcntr/messageforums-app/src/webapp/jsp/dfReviseForumSettingsAttach.jsp [hunk 1]
```diff
@@ -109,28 +109,33 @@
 					format: charRemFormat
 				 });
 				 updateGradeAssignment();
-			 });				 
+
+                if($('#revise\\:createForumsForGroups\\:1').prop('checked') == true) {
+                    setAutoCreatePanel($('#revise\\:createForumsForGroups'));
```
Comment (adrianfish) [inline msgcntr/messageforums-app/src/webapp/jsp/dfReviseForumSettingsAttach.jsp]: setAutoCreatePanel doesn't seem to even use the jquery object passed in. You could perhaps just remove that from the function signature and remove these selectors in the calls. I don't even know why it's called setAutoCreatePanel. It is forums, though :)

File: msgcntr/messageforums-app/src/webapp/jsp/dfReviseForumSettingsAttach.jsp [hunk 2]
```diff
@@ -109,28 +109,33 @@
 					format: charRemFormat
 				 });
 				 updateGradeAssignment();
-			 });				 
+
+                if($('#revise\\:createForumsForGroups\\:1').prop('checked') == true) {
```
[the remark this answers is not in the document]
Comment (mpellicer) [reply msgcntr/messageforums-app/src/webapp/jsp/dfReviseForumSettingsAttach.jsp]: You don't like jQuery eh? Understood.

File: msgcntr/messageforums-app/src/webapp/jsp/dfReviseForumSettingsAttach.jsp (hunk 1 above)
Comment (mpellicer) [reply msgcntr/messageforums-app/src/webapp/jsp/dfReviseForumSettingsAttach.jsp]: Yeah, can be removed.

**Comment by mpellicer** [comment]
@adrianfish changes implemented.

**Comment by mpellicer** [comment]
@adrianfish changes implemented.

**Pull request merged**